vita3k/util/include/util/tracy.h

```diff
@@ -110,8 +123,12 @@ inline std::string to_debug_str(const MemState &mem) {
 #else
 #define TRACY_FUNC(name, ...) __TRACY_FUNC(tracy_module_name, name, ##__VA_ARGS__, , , , , , , , )
 #define TRACY_FUNC_M(module_name_var, name, ...) __TRACY_FUNC(module_name_var, name, ##__VA_ARGS__, , , , , , , , )
+#define TRACY_FUNC_COMMANDS(name) __TRACY_FUNC_COMMANDS(name)
+#define TRACY_FUNC_COMMANDS_SET_STATE(name) __TRACY_FUNC_COMMANDS_SET_STATE(name)
```

Comment:
There is no __VA_ARGS__ in TRACY_FUNC_COMMANDS, so workaround for __VA_ARGS__ is not needed (and also TRACY_FUNC_COMMANDS... should be defined in both branches of #if)

Reply:
does last push fix it?

Reply:
yes.